py2hwsw/lib/hardware/fifo/iob_bfifo/iob_bfifo.py

```diff
@@ -5,21 +5,328 @@
 
 def setup(py_params_dict):
     attributes_dict = {
-        "generate_hw": False,
+        "generate_hw": True,
+        "confs": [
+            {
+                "name": "DATA_W",
+                "descr": "Data width",
+                "type": "P",
+                "val": "21",
+                "min": "NA",
+                "max": "NA",
+            },
+            {
+                "name": "BUFFER_SIZE",
```

Comment:
Why was this moved from localparam to normal param? We try to avoid derived parameters since they get flagged in linting

Reply:
yes, we should create a type L (localparam) for iob_confs. In the meantime, we can use them in snippets